doc/stdlib/base.rst

```diff
@@ -156,6 +156,10 @@ Getting Around
 
    This is only needed if your module depends on a file that is not used via ``include``\ . It has no effect outside of compilation.
 
+.. function:: apropos(string)
+
+   .. Docstring generated from Julia source
+
```

Comment:
Totally agree, my take is that we should be able to document everything, whether it is exported,  meant to be "unexported but public" (like things in Pkg) or even "unexported meant to be private", no matter where it is in modules.